common/map_file/launch/map_param_loader.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+ <arg name="file_name"/>	
+ <node pkg="map_file" type="map_param_loader" name="map_param_loader">  
+  <rosparam param = "file_name">$(arg file_name)</rosparam>
```

Comment:
If you are using setting a parameter from a launch file argument it is better to use the param tag. rosparam is for loading parameters from yaml files. So this would change to
```
<param name="file_name" value="$(arg file_name)"/>
```